src/amt/ConnectedDevice.ts

```diff
@@ -219,11 +219,11 @@ export class ConnectedDevice {
 
   async getAuditLog (startIndex: number): Promise<any> {
```

Comment:
Lets create a type for the return: 
```
export interface CIM_SoftwareIdentityResponse{
CIM_SoftwareIdentity: CIM_SoftwareIdentity
}
```

Reply:
I created a new return type and added it to src/amt/models/amt_models.ts:

```
export interface AMT_AuditLog_ReadRecords {
  ReadRecords_OUTPUT: {
    TotalRecordCount: string
    RecordsReturned: string
    EventRecords: string[]
    ReturnValue: string
  }
}
```